onnxruntime/core/providers/mkldnn/subgraph/subgraph.h

```diff
@@ -48,8 +48,12 @@ struct Subgraph {
     std::vector<std::string> outputs;
     std::vector<std::string> outputs_as_input_other_node;
     std::vector<onnxruntime::NodeIndex> subgraph_node_indexes;
+    int subgraph_index = 0;
 
-	void Reset() {
+    SubgraphVariables() {
```

Comment:
why is this needed? i thought we got rid of it in https://github.com/microsoft/onnxruntime/pull/1740